sdk/pyproject.toml

```diff
@@ -3,17 +3,17 @@ requires = ["setuptools >= 61.0"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "basyx-python-framework"
+name = "basyx"
```

Comment:
I guess we should call this something like `basyx-python-framework-base` or `basyx-python-framework-sdk`, otherwise it might be very confusing. 
Let's discuss naming options inthe next meeting.